tox.ini

```
@@ -15,7 +23,11 @@ deps=
     django20: django==2.0.*
     django21: django==2.0.*
     django22: django==2.0.*
-    django30: django==3.0b1
+    django30: django==3.0.*
+    django31: django==3.1.*
+    django32: django==3.2.*
+    django40: django==4.0.7
+    django41: django==4.1
```

Do we need to keep deprecated django versions here?
```suggestion
    django41: django==4.1
    django42: django==4.2
```